### DISCUSSION TITLE
请问内核日志和app日志在同一个文件吗？

### DISCUSSION THREAD

**Discussion by kty00** [clash-verge-rev/clash-verge-rev#2013] [Q&A]
![image](https://github.com/user-attachments/assets/4605da9a-0c22-4dc3-9628-bf10080e8252)
是不是红色框的是 mihomo 内核日志？也就是 service 目录。然后，蓝色框的是 app 日志？

**Comment by iwalke** [comment]
没分开吧？可读性不高，应该区分以下，比如 kernel-log，app-log 这样

**Reply by iwalke** [reply]
@wonfen 怎么样
